src/timer.rs

```diff
@@ -269,16 +307,91 @@ macro_rules! hal {
     }
 }
 
+hal! {
+    TIM2: (tim2, pclk1_tim, dbg_tim2_stop),
+    TIM3: (tim3, pclk1_tim, dbg_tim3_stop),
+}
+
 #[cfg(any(
-    feature = "stm32f100",
-    feature = "stm32f103",
+    feature = "stm32f100", 
+    feature = "stm32f103", 
+    feature = "stm32f105",
 ))]
 hal! {
     TIM1: (tim1, pclk2_tim, dbg_tim1_stop),
 }
 
+#[cfg(any(
+    feature = "stm32f100",
+    feature = "stm32f105", 
+    feature = "high",
+))]
+hal! {
+    TIM6: (tim6, pclk2_tim, dbg_tim6_stop),
+}
+
+#[cfg(any(
+    all(
+        feature = "high",
+        any(
+            feature = "stm32f101", 
+            feature = "stm32f103", 
+            feature = "stm32f107",
+        ),
+    ),
+    any(
+        feature = "stm32f100",
+        feature = "stm32f105",
+)))]
+hal! {
+    TIM7: (tim7, pclk2_tim, dbg_tim7_stop),
+}
+
+#[cfg(feature = "stm32f100")]
+hal! {
+    TIM15: (tim15, pclk2_tim, dbg_tim15_stop),
+    TIM16: (tim16, pclk2_tim, dbg_tim16_stop),
+    TIM17: (tim17, pclk2_tim, dbg_tim17_stop),
+}
+
+#[cfg(feature = "medium")]
 hal! {
-    TIM2: (tim2, pclk1_tim, dbg_tim2_stop),
-    TIM3: (tim3, pclk1_tim, dbg_tim3_stop),
     TIM4: (tim4, pclk1_tim, dbg_tim4_stop),
 }
+
+#[cfg(feature = "high")]
+hal! {
+    TIM5: (tim5, pclk2_tim, dbg_tim5_stop),
+}
+
+#[cfg(all(
+    feature = "stm32f103", 
+    feature = "high",
+))]
+hal! {
+    TIM8: (tim8, pclk2_tim, dbg_tim8_stop),
+}
+
+//TODO: restore these timers once stm32-rs has been updated
```

Comment:
`stm32-rs` 0.9 released and we now use it or you mean something other?

Reply:
That refers to the missing dbg_tim(9 to 14)_stop fields on dbgmcu. They haven't been implemented in stm32-rs yet.